### PR TITLE
Rt 132950 20180308

### PR DESCRIPTION
**Pull request by jkeenan**

### CODE REVIEW

File: cpanfile [hunk 1]
```diff
@@ -43,6 +43,7 @@ requires 'Test::Builder';
 requires 'Test::More';
 requires 'Type::Tiny', '1.000006';
 requires 'URI::Escape';
+requires 'XML::Simple';
```
Comment (Grinnz) [inline cpanfile]: What's the purpose of this change?

**Comment by SysPete** [comment]
This is not a Perl BBC but an existing known issue: https://github.com/PerlDancer/Dancer2/issues/1435. I'll add a comment to the RT ticket.

**Comment by jkeenan** [comment]
On 03/08/2018 12:32 PM, Dan Book wrote:
> *@Grinnz* commented on this pull request.
> 
> ------------------------------------------------------------------------
> 
> In cpanfile 
> <https://github.com/PerlDancer/Dancer2/pull/1437#discussion_r173232139>:
> 
>> @@ -43,6 +43,7 @@ requires 'Test::Builder';
>   requires 'Test::More';
>   requires 'Type::Tiny', '1.000006';
>   requires 'URI::Escape';
> +requires 'XML::Simple';
> 
> What's the purpose of this change?
> 

Once I updated my Dancer2 repository, then installed Perl 5 blead and 
Dist-Zilla's enormous list of prerequisites, I called:

/home/jkeenan/testing/blead/bin/dzil test --no-author

and got test failures:

#####
t/serializer_xml.t ..................................... Can't locate 
XML/Simple.pm in @INC (you may need to install the XML::Simple module) 
(@INC contains: 
/home/jkeenan/gitwork/zzzothers/Dancer2/Dancer2-0.200003/blib/lib 
/home/jkeenan/gitwork/zzzothers/Dancer2/Dancer2-0.200003/blib/arch 
/home/jkeenan/testing/blead/lib/perl5/site_perl/5.27.10/x86_64-linux 
/home/jkeenan/testing/blead/lib/perl5/site_perl/5.27.10 
/home/jkeenan/testing/blead/lib/perl5/5.27.10/x86_64-linux 
/home/jkeenan/testing/blead/lib/perl5/5.27.10 .) at t/serializer_xml.t 
line 9.
BEGIN failed--compilation aborted at t/serializer_xml.t line 9.
t/serializer_xml.t ..................................... Dubious, test 
returned 2 (wstat 512, 0x200)
No subtests run
#####

That led to my first p.r.:
https://github.com/PerlDancer/Dancer2/pull/1436

Working in that branch, I then began to examine the test failure 
reported in RTR #132950, which led to the second pull request.

So, please consider p.r. 1436.

Thank you very much.
Jim Keenan

**Comment by Grinnz** [comment]
t/serializer_xml.t does not seem to exist. Are you sure your git checkout is clean?

**Comment by veryrusty** [comment]
Thanks for the pull request @jkeenan.

The issue with HTTP::XSCookies has been addressed upstream. As such, there is no need to consider merging this. Closing the PR.

As @Grinnz notes, its best to keep unrelated patches in seperate branches in the future.
